ExamPlatform/Models/Candidate.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Web;
+
+namespace ExamPlatform.Models
+{
+    public class Candidate
+    { 
+        public int CandidateId { get; set; }
+
+        public string Name { get; set; }
+
+        public string Email { get; set; }
+
+        [Display(Name = "Phone Number")]
```

Comment:
add Model Validation Using Data Annotations like max length, email validation, min length, regex validation 
follow https://docs.microsoft.com/en-us/aspnet/mvc/overview/older-versions-1/models-data/validation-with-the-data-annotation-validators-cs